### ISSUE TITLE
Exposure 137368 should be marked as bad due to missing guide info

### ISSUE THREAD

**Issue by djschlegel** [desihub/desisurveyops#55]
The guider crashed on exposure 137368 on tile 10589 on 2022-05-30.  Night log states:
GUIDE5 GFA image size error followed by second failed automatic revere attempt. Loss of guiding mid-exposure.; Action: Stopped exposure and read out data for DARK tile 10589 exposure [137368] once guiding was confirmed as lost.

This results in no guider cube being written for the exposure.  We've been marking such exposures as bad, since we lose the metadata such as the guider-computed effective times.  The last such instance was exposure 133140 on 20220503, triggered from a failure of GUIDE7.

**Comment by akremin** [comment]
Yes this was marked as bad and removed before the 10am updates that morning, so everything for this exposure is taken care of.
